src/application/dockpanel.css

```diff
@@ -43,11 +39,6 @@
 }
 
 
-#jp-main-dock-panel .p-TabBar-tab.jp-mod-current {
-  border-top: var(--jp-border-width) solid var(--jp-brand-color1);
-}
-
-
 #jp-main-dock-panel {
-  padding: 4px;
+  padding: 5px;
```

Comment:
In general I am trying to do that, but tried it at 4px for a long time and it just felt too crowded...there are a few other places in the notebook where we are using 5px increments because of this as well...